### ISSUE TITLE
Wishes for xmas specials

### ISSUE THREAD

**Issue by janexx** [unknown-horizons/unknown-horizons#2554]
Hey everyone, I would like to have a special xmas-edition of UH or at least a small easteregg for christmas. Nothing big, just replacing/adding some graphics. What do you all think about that? What would you like to see in UH? 
Examples:  
- a christmastree at the marketplace
- some gifts appearing randomly somewhere (would need additional code)
- ?

**Comment by LinuxDonald** [comment]
What about an future that we automatically activate from December 1 to 31 Xmas setting in UH?

**Comment by janexx** [comment]
Good idea @LinuxDonald. Woulde this cause much work?
OK two people seem to like the idea, with me 3. :) Maybe I start with the christmas tree. And when I get more hearts, I will do more christmas stuff. :)

**Comment by LinuxDonald** [comment]
I dont know about the work but i asked MasterOfJoker allready and he dont have time at the moment. Maybe @squiddy could do this but i dont know :(

**Comment by janexx** [comment]
How it could look like...
![xmas_market_citizens](https://cloud.githubusercontent.com/assets/6216898/20677304/ed90e4b2-b592-11e6-8453-2299a91b7289.png)

**Comment by LinuxDonald** [comment]
That's nice :-)

**Comment by Jeeppler** [comment]
Was this ever implemented? The Christmas tree idea is good.

**Comment by LinuxDonald** [comment]
No. We not have enough members todo this. And I think it will not happen for this x-mas. Maybe next one.
